build/Dockerfile.openshift

```diff
@@ -0,0 +1,18 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+WORKDIR /go/src/github.com/nmstate/kubernetes-nmstate
+COPY . .
+RUN GO111MODULE=on go build --mod=vendor -o build/_output/bin/kubernetes-nmstate cmd/manager/main.go
+
+FROM centos:8
```

Comment:
This should probably be the ubi8-minimal like the operator if it's for OpenShift.

Reply:
Done